cmd/plugins.go

```diff
@@ -168,9 +170,18 @@ var pluginTidyCmd = &cobra.Command{
 		return nil
 	},
 	Run: func(cmd *cobra.Command, args []string) {
+		var (
+			err error
+			mod *modfile.File
+		)
 		assertInFlutterProject()
 		assertHoverInitialized()
 
+		if mod, err = modx.Open(build.BuildPath); err != nil {
```

Comment:
this chunk of work lets us overwrite the file once after all the mutations have been applied.

Reply:
The `mod` var feels scoped to the if block, but isn't. That is confusing.
Please write this as:
```go
mod, err := modx.Open(build.BuildPath)
if err != nil {
    ...
```
And remove the `var( err, mod )` block above.

Reply:
its not scoped to the if block. its used throughout this entire function.

Reply:
> its not scoped to the if block. its used throughout this entire function

Yes, that's why having it inlined in the `if` statement is confusing.